### PR TITLE
A few tweaks to the SSO docs for Google Workspace (SAML)

### PR DESCRIPTION
**Pull request by yob**
We recently re-discovered that Buildkite is listed in the preconfigured app catalog in Google Workspace, so I've updated the docs to recommend that path.

It doesn't simplify the process like I hoped it would, but one day we hope to convince google to enable SCIM on our app so it'll be good to encourage folks to use it.

While testing the revised process out, I also discovered that it can sometimes take many hours for changes to the SAML app to take effect, and while you wait the errors are confusing. I've added a notice to warn future travellers.

Finally, I added a short note at the top that describes how users can login to Buildkite once the process is complete. It's kind of neat - a Buildkite icon appears in the same menu as gmail/calendar/youtube/etc.

### CODE REVIEW

**Comment by karensawrey** [comment]
Thanks a million for the added info, @yob! This should be super useful. 
The PR is still marked as "draft" - is this still a work in progress or would you like me to tweak a couple of things to pacify the linter & merge?

File: pages/integrations/sso/google_workspace_saml.md.erb [hunk 1]
```diff
@@ -8,6 +8,8 @@ To complete this tutorial, you will need admin privileges for both Google Worksp
   <p>You can also set up SSO providers manually with GraphQL. See the <a href="/docs/integrations/sso/sso-setup-with-graphql">SSO Setup with GraphQL Guide</a> for detailed instructions and code samples.</p>
 </div>
 
+Once this process is complete, users can sign into Buildkite by clicking the icon in the "9 dots" applications menu of the top bar available on many Google products.
```
Comment (yob) [inline pages/integrations/sso/google_workspace_saml.md.erb:11]: I considered putting a screenshot here, but... the one I took on linux looks a bit ugly:

google-initiated-login

I don't think it's critical. The info added in this PR is still an improvement, even without the image.

Comment (karensawrey) [reply pages/integrations/sso/google_workspace_saml.md.erb:11]: Got it! So will you be OK with me going over your PR + possibly adding a prettier screenshot & merging, @yob? No time pressure, just want to figure out what to do with this piece of docs whenever you're ready.

**Comment by yob** [comment]
Thanks @karensawrey!

I'm in your hands. I'm confident the changes here are technically accurate, and I'm happy with whatever process you think is best to land this in prod.

I can make any changes you'd like me too, or you can run with it and merge when you're ready ❤️

**Review by karensawrey — APPROVED** [review]

File: pages/integrations/sso/google_workspace_saml.md.erb [hunk 2]
```diff
@@ -8,6 +8,10 @@ To complete this tutorial, you will need admin privileges for both Google Worksp
   <p>You can also set up SSO providers manually with GraphQL. See the <a href="/docs/integrations/sso/sso-setup-with-graphql">SSO Setup with GraphQL Guide</a> for detailed instructions and code samples.</p>
 </div>
 
+As a result of this tutorial, you will enable users to sign into Buildkite by clicking the Buildkite icon in the Google Apps menu. You can find this menu by clicking the 9-dot "waffle" icon.
```
Comment (plaindocs) [inline pages/integrations/sso/google_workspace_saml.md.erb:11]: Let's be more direct about this

> After following these steps, your Google Workspace users will be able to sign into Buildkite using their google account. 

the Google Apps menu is incidental, let's move that to end of the tutorial. Do you agree @karensawrey ?

**Review by plaindocs — COMMENTED** [review]
The rest of this looks great, thanks @yob @karensawrey

File: pages/integrations/sso/google_workspace_saml.md.erb (hunk 2 above)
Comment (karensawrey) [reply pages/integrations/sso/google_workspace_saml.md.erb:11]: I do agree! Your suggestion of "here is a screenshot of what you should be seeing now" trumps the current logic of "here's a screenshot for you to know what you're trying to accomplish".

File: pages/integrations/sso/google_workspace_saml.md.erb [hunk 3]
```diff
@@ -8,6 +8,8 @@ To complete this tutorial, you will need admin privileges for both Google Worksp
   <p>You can also set up SSO providers manually with GraphQL. See the <a href="/docs/integrations/sso/sso-setup-with-graphql">SSO Setup with GraphQL Guide</a> for detailed instructions and code samples.</p>
 </div>
 
+After following these steps, your Google Workspace users will be able to sign into Buildkite using their Google account.
```
Comment (plaindocs) [inline pages/integrations/sso/google_workspace_saml.md.erb:11]:
```suggestion
After following these steps, your Google Workspace users will be able to sign in to Buildkite using their Google account.  
```
my mistake

**Review by plaindocs — APPROVED** [review]

**Comment by yob** [comment]
Thanks all :heart:

**Pull request merged**